### ISSUE TITLE
[DUBTE] Ranking evaluation (Diapo 37)

### ISSUE THREAD

**Issue by gerardkm94** [gdsa-upc/gdsa-suport#12]
A la avaluació del rànquing, hi ha dos paràmetres d'entrada, un és el directori amb el rànquing, i l'altre el Ground Truth (anotacions del training set i del Val/test set), aquestes anotacions, intueixo que són les originals no? és a dir les correctes i no les generades aleatòriament.

Gràcies

**Comment by amaiasalvador** [comment]
Sí, les anotacions que necessites són les de veritat (és a dir, les dels fitxers _annotation.txt_ que us hem donat per cada conjunt), ja que vols avaluar la qualitat dels rankings que hauràs generat.  Per cada imatge query del conjunt de validació tindràs un ranking que serà un llistat de les IDs de les imatges del conjunt d'entrenament ordenades aleatòriament (recorda però que les imatges de la classe "desconegut" del conjunt d'imatges de validació no les has d'utilitzar com a queries).

Per avaluar aquests rankings, necessitaràs:

(1) Els rankings !

(2) Les anotacions del conjunt de validació, per saber quina és la classe de cadascuna de les imatges query. 

(3) Les anotacions del conjunt d'entrenament. Per calcular l'Average Precision del ranking, hauràs de comparar la classe (ground truth) de cada imatge query amb la classe de les imatges del seu ranking.
